.github/workflows/dashwallet.yml

```diff
@@ -15,7 +15,11 @@ jobs:
     - name: Get build number from run id
       run: |
           echo "build_number=$((70000))" >> $GITHUB_ENV
-    
+
+    - name: Get the support email
+      env:
+        SUPPORT_EMAIL: "${{ secrets.INTERNAL_SUPPORT_EMAIL }}"
```

Comment:
This was found to work, but should I use the same method as line 17 to get an environment variable "registered"

Reply:
@HashEngineering I don't think so, I was using echo to calculate build number (offset + run_num), so it might actually not be needed anymore here. The way you set env variable here should be fine.